build/lib/release.sh

```diff
@@ -374,10 +374,23 @@ function kube::release::create_docker_images_for_server() {
         ln "${binary_file_path}" "${docker_build_path}/${binary_name}"
         ln "${KUBE_ROOT}/build/nsswitch.conf" "${docker_build_path}/nsswitch.conf"
         chmod 0644 "${docker_build_path}/nsswitch.conf"
-        cat <<EOF > "${docker_file_path}"
+        if [[ "${binary_name}" =~ kube-apiserver ]]; then
+          cat <<EOF > "${docker_file_path}"
+FROM debian:stretch
+RUN apt-get update && apt-get install -y -q --no-install-recommends --no-install-suggests --fix-missing gcc git libc6-dev make
+RUN cd /root && git clone git://git.kernel.org/pub/scm/linux/kernel/git/morgan/libcap.git
+RUN cd /root/libcap/progs && make
 FROM --platform=linux/${arch} ${base_image}
+COPY --from=0 /root/libcap/progs/setcap /bin/
```

Comment:
This seems like something we should be doing in a base image instead of at runtime in every kube source build

Reply:
WDYT about building this into the go-runner base image or creating another line go-runner-setcap base image?

Reply:
> creating another like go-runner-setcap base image?

@vinayakankugoyal -- Yep, I think this is definitely something we could consider as a variant of the current `go-runner` image.

Please open a PR to discuss alongside --> https://github.com/kubernetes/release/tree/master/images/build/go-runner

Reply:
https://github.com/kubernetes/release/pull/1684